### DISCUSSION TITLE
GUI怎么启动啊

### DISCUSSION THREAD

**Discussion by XIKONGSAKET** [CHNZYX/Auto_Simulated_Universe#181] [Q&A]
虽然我知道用bat可以用并且我用了几个星期了。但是，双击GUI.EXE就从来没有运行出来任何窗口过。
我搞不明白为啥。明明bat运行是可以运行的说明环境没毛病啊。但是GUI就是从来都没弹出来过
究竟是为什么呢？而且主页的介绍也从来没有过GUI的截图，我不确定正确运行的GUI长啥样。。
另外，最近总是由于mylib什么的运行不了，issue里面也说了。xdm有啥解释嘛。CHATGPT看完报错说没装这个包，但是检查了很多次是装了的。

**Comment by wyc990122** [comment]
上周我的gui.exe还可以正常启动，这周更新后的gui.exe双击后也无法启动了。希望能得到解决

**Comment by CHNZYX** [comment]
可能受一个bug影响，现在修复了，可以更新试试

**Reply by FG007** [reply]
昨天我试了也是一样的，更新之后就没法用了
